book/src/SUMMARY.md

```diff
@@ -87,3 +90,5 @@
   * [Rent](implemented-proposals/rent.md)
   * [Durable Transaction Nonces](implemented-proposals/durable-tx-nonces.md)
   * [Validator Timestamp Oracle](implemented-proposals/validator-timestamp-oracle.md)
+  * [Commitment](implemented-proposals/commitment.md)
+  * [Snapshot Verification](implemented-proposals/snapshot-verification.md)
```

Comment:
Implemented?

Reply:
I didn't move these documents, I was relying on their location in the tree.

@ryoqun for snapshot verification
@carllin for commitment